src/index.jsx

```diff
@@ -34,6 +36,11 @@ subscribe(APP_INIT_ERROR, (error) => {
 });
 
 initialize({
+  config: () => {
+    mergeConfig({
+      STUDIO_BASE_URL: process.env.STUDIO_BASE_URL || null,
```

Comment:
What do we need the Studio URL for? From a quick glance, I thought all the nav points we are adding are for places in the LMS and not the CMS?

Reply:
Good point. This was needed when we had the full instructor page toolbar nav. Now that this is only the course navigation this is no longer needed.